test/cache_test.rb

```diff
@@ -150,6 +156,14 @@ def test_fragment_fetch_with_virtual_associations
       assert_equal({ place: 'Nowhere' }, ActionController::Base.cache_store.fetch(@location.cache_key))
     end
 
+    def test_fragment_cache_with_inheritance
+      inherited = render_object_with_cache(@role, serializer: InheritedRoleSerializer)
+      base = render_object_with_cache(@role)
+
+      assert_includes(inherited.keys, :special_attribute)
```

Comment:
Not sure what you mean here, should I not use assert_includes? Or am I misunderstanding this method? http://apidock.com/ruby/v1_9_3_392/MiniTest/Assertions/assert_includes